packages/components/src/tooltip/index.tsx

```diff
@@ -44,8 +46,29 @@ function Tooltip( props: TooltipProps ) {
 		}
 	}
 
+	const DEFAULT_PLACEMENT = 'bottom';
+
+	// Compute tooltip's placement:
+	// - give priority to `placement` prop, if defined
+	// - otherwise, compute it from the legacy `position` prop (if defined)
+	// - finally, fallback to the DEFAULT_PLACEMENT.
+	let computedPlacement;
+	if ( placement !== undefined ) {
+		computedPlacement = placement;
+	} else if ( position !== undefined ) {
+		computedPlacement = positionToPlacement( position );
+	}
+	computedPlacement = computedPlacement || DEFAULT_PLACEMENT;
+
+	if ( position !== undefined ) {
+		deprecated( '`position` prop in wp.components.tooltip', {
+			since: '6.4',
+			alternative: '`placement` prop',
+		} );
+	}
```

Comment:
We have this `if ( position !== undefined )` condition above already, so perhaps we can just move the `deprecated()` call there, instead of duplicating the check?

Reply:
I borrowed [the logic from the same changes made to URLPopover](https://github.com/WordPress/gutenberg/pull/44391/files). Do you think it's worth following up there as well with your suggestions?

Reply:
Yes. Feel free to follow up in another PR, although it's not a huge deal, just a minor code style preference.